clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/ops/BulkUpsertEntityTagsAtomicOperation.java

```diff
@@ -246,8 +246,16 @@ private static void addTagMetadata(Date now, EntityTags entityTags) {
     entityTags.getTags().forEach(tag -> entityTags.putEntityTagMetadata(tagMetadata(tag.getName(), now)));
   }
 
-  private static AccountCredentials lookupAccountCredentials(AccountCredentialsProvider accountCredentialsProvider,
-                                                             String entityRefAccountId) {
+  private static AccountCredentials lookupAccountCredentialsByAccountName(AccountCredentialsProvider accountCredentialsProvider,
```

Comment:
Yea, agreed.  I did like the explicit error message, so I'll add a parameter to specify that.